pkg/plugins/golang/v3/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -83,9 +83,11 @@ type {{ .Resource.Kind }}Reconciler struct {
 	Scheme *runtime.Scheme
```

Comment:
**General guidance about the PRs**

a) Ensure that you run make generate to update the samples
b) Use the mojo in the title of your PR.
See: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#pr-process (in this case would be `:bug:`)
c) You need to assign CLA: https://github.com/kubernetes-sigs/kubebuilder/pull/2430#issuecomment-984301237 at the first time.